src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -1,17 +1,32 @@
 package racingcar.domain
 
 class RacingCar(
-    var position: Int = 0
+    private val name: String,
+    private var position: Int = 0
 ) {
-    companion object {
-        private const val MOVE_POSSIBLE_NUMBER = 4;
+    init {
+        require(name.length <= 5) {
+            throw IllegalArgumentException("자동차 이름은 5자 까지 사용 가능 합니다. 입력하신 차의 이름은 : $name 입니다.")
+        }
```

Comment:
```suggestion
        require(name.length <= 5) {
            "자동차 이름은 5자 까지 사용 가능 합니다. 입력하신 차의 이름은 : $name 입니다."
        }
```

이렇게도 가능해보입니다! 왜 IllegalArgumentException이 지워졌을까요?

Reply:
`require` 안에서 `IllegalArgumentException`을 처리하고 있네요